README.md

```
@@ -2,190 +2,113 @@
-[![Docs](https://img.shields.io/badge/docs-online-0284c7.svg?style=flat-square)](https://pymodel.github.io/pythinker-code/)
-[![Homepage](https://img.shields.io/badge/home-code.pythinker.com-ec4899.svg?style=flat-square)](https://code.pythinker.com)
-
-<br />
-
-<a href="https://code.pythinker.com">Website</a> &nbsp;·&nbsp;
-<a href="#quick-start">Quick Start</a> &nbsp;·&nbsp;
-<a href="#features">Features</a> &nbsp;·&nbsp;
-<a href="#ide-integration-via-acp">IDE Integration</a> &nbsp;·&nbsp;
-<a href="#mcp-tooling">MCP</a> &nbsp;·&nbsp;
-<a href="#architecture">Architecture</a> &nbsp;·&nbsp;
+### A coding agent you can run as a desktop app
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)